templates/microprofile-tomee/files/.platform.app.yaml

```diff
@@ -0,0 +1,41 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+name: app
+
+# The runtime the application uses.
+type: "java:8"
+
+disk: 1024
+
+# The hooks executed at various points in the lifecycle of the application.
+hooks:
+    build: mvn -Dhttp.port=8888 -DskipTests clean install tomee:exec
+
+mounts:
+
+    '/.distribution':
+        source: local
+        source_path: tomee_source
+
+    '/app':
```

Comment:
same thing, the whole `/app` needs write access?

Reply:
Yes, this vendor is even worse on that approach because it needs the second folder to write on it.